### PR TITLE
DDF-2309 Trivial typo fix in platform documentation

### PR DESCRIPTION
**Pull request by bradh**
#### What does this PR do?
Fixes a simple typo in the extension docs.

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@ricklarsen 

#### How should this be tested?
Visual inspection should be enough.

#### What are the relevant tickets?
Hopefully small enough not to need one.

#### Screenshots (if appropriate)
See http://codice.org/ddf/extending.html#_developing_action_components_action_framework

#### Checklist:
- [X ] Documentation Updated
- [ N/A] Update / Add Unit Tests
- [ N/A] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_platform-contents/extending-platform-contents.adoc [hunk 1]
```diff
@@ -43,8 +43,8 @@ These are available for use by third party developers but their versions can cha
...
 That feature does not have to be coupled to a REST endpoint to work, nor does it have to depend on a specific implementation to get a link.
-In reality, the feature does not identify how the link is generated, but it does identify whether link works or does not work when retrieving the intended entry's metadata. \Instead of creating its own mechanism or adding an unrelated feature, it could use the Action Framework to query out in the OSGi container for any service that can provide a link.
-This does two things: it allows the feature to be independent of implementations, and it encourages reuse of common services. 
+In reality, the feature does not identify how the link is generated, but it does identify whether link works or does not work when retrieving the intended entry's metadata. Instead of creating its own mechanism or adding an unrelated feature, it could use the Action Framework to query out in the OSGi container for any service that can provide a link.
+This does two things: it allows the feature to be independent of implementations, and it encourages reuse of common services.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_platform-contents/extending-platform-contents.adoc]: +1

**Comment by ricklarsen** [comment]
Adding @shaundmorris to click the `merge` button

**Comment by coyotesqrl** [comment]
@bradh I've updated the title on your PR to associate it with an open documentation ticket. Ideally, we'd like you to rebase so you can rewrite your commit message so it can more easily be tracked in the future. In a few months when the time comes to cut a major release it's much simpler to compile a list of enhancements and bug fixes if commits can quickly be scanned and correlated back to issues.

I suggest you take a look at the [DDF Contribution Guidelines](https://codice.atlassian.net/wiki/display/DDF/Community+Contributions) when you have a chance and in particular take a look at the best practices for [DDF Commit messages](https://codice.atlassian.net/wiki/display/DDF/Git+Commit+Message+Guidelines).

File: distribution/docs/src/main/resources/_contents/_platform-contents/extending-platform-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_platform-contents/extending-platform-contents.adoc]: Just recreated this fix in the PR for https://github.com/codice/ddf/pull/1217

**Comment by bradh** [comment]
Too much work to rebase given its been redone.

**Pull request closed without merging**